src/main/java/jalau/at18/azul/Tiles.java

```diff
@@ -0,0 +1,21 @@
+package jalau.at18.azul;
+
+public enum Tiles {
+    RED('R'),
+    YELLOW('Y'),
+    DARK('D'),
+    BLUE('B'),
+    WHITE('W'),
+    FIRST_PLAYER('F');
+
+    private final char colorTile;
```

Comment:
Maybe we don't need this attribute, you could fill the pyramid directly with enum values instead of chars. Say for example, if you have pyramid class with a list of tiles, you can have
```
List<Tile> tiles
```
instead of
```
List<Character> tiles
```

Reply:
We decide as a group to put a character in the attribute of the enum values, because for fill the pyramid the char character only occupied 1 space in the pyramid, instead on put the whole name as RED or YELLOW for example

Reply:
You would be actually putting 1 object (tile enum instance) and it could be more readable. I still think that this would be better to handle it with your enum directly instead of Character, but we can go ahead with your approach and see how it goes.

Reply:
Thanks for the advice, i'm going to discuss this with my group to define wich way it's better :)